src/python/bcc/table.py

```diff
@@ -412,9 +412,37 @@ def __init__(self, *args, **kwargs):
         raise Exception("Unsupported")
 
 class StackTrace(TableBase):
+    MAX_DEPTH = 127
```

Comment:
PERF_MAX_STACK_DEPTH is highly unlikely to change. All sorts of abi concerns will pop up, including increase in size of perf.data files, since 127 is also default for 'perf record -g'